projects/plugins/social/src/js/components/icon-text-section/styles.module.scss

```diff
@@ -0,0 +1,58 @@
+.column {
+	grid-column: 1 / -1;
+	display: grid;
+	grid-template-columns: auto 1fr;
+	grid-template-rows: repeat( 2, auto );
+	column-gap: calc( var( --spacing-base ) * 3 );
+	row-gap: calc( var( --spacing-base ) * 2 );
+
+	@supports ( display: grid ) {
+		> * + * {
+			margin-top: 0;
+		}
+	}
```

Comment:
For this to do anything, I think you also want to add some margin in case the browser doesn't support grid.

```css
> * + * {
	margin-top: calc( var( --spacing-base ) * 2 );
}

@supports ( display: grid ) {
	> * + * {
		margin-top: 0;
	}
}
```

You can simulate the non-grid behavior by toggling it off in the CSS Inspector in your browser.

Reply:
Done!